### DISCUSSION TITLE
Add option to display more id3 tags

### DISCUSSION THREAD

**Discussion by fpunktk** [PaulWoitaschek/Voice#1414] [Ideas]
Not a bug, but a feature request: 
Voice seems to display the "album" id3-tag only. For audiobooks this seems to be okay, but for podcasts all episodes look the same (because they usually all have the same "album" tag). It would be great if you could add an option to display the "title" and "artist" id3 tag, too.
Issue #934 seems to be about the same problem. Displaying the filename could also solve it, but I think displaying the id3 tags is a nicer solution ;-)

**Comment by denniz-j** [comment]
The problem does not only affect Podcasts. Audiobooks from Librivox seem to have the same problem. Different chapters from the one book all display the same information, and it is difficult to know which chapter is which. For example Charles Darwin's "The Voyage of the Beagle" from [Librivox > The Voyage of the Beagle](https://librivox.org/the-voyage-of-the-beagle-by-charles-darwin/) has all it's chapters rendered by Voice as in the following image...

![Screenshot_20200523-211052v2](https://user-images.githubusercontent.com/26517622/82732221-104fe180-9d4b-11ea-990e-e35772adfd67.png)

Displaying the "title" id3 tag as well as the tags currently displayed by Voice would be a great help when listening to Librivox audiobooks.

And thanks for a great app!

(PS. June 1st 2020.  I'm new to audiobooks, and just discovered, after I submitted the above comment, that Librivox.org makes available .m4b format versions of their narrations as well as the .mp3 collections for each book. These .m4b versions overcome the problems of navigation that the 'raw' .mp3 files have in the Voice app.)

**Comment by KL4RKS** [comment]
My MP3 audiobooks also have a book description in the "comment" tag. I'd love for that to display somewhere in Voice.
